composer.json

```diff
@@ -33,6 +33,9 @@
         "phpstan/phpstan-shim": "^0.9.2",
         "phpunit/phpunit": "^7.0"
     },
+    "conflict": {
+        "phpspec/prophecy": "<1.7.5"
+    },
```

Comment:
You are blocking potential users of prophecy < 1.7.5 even if they don't intend to test doctrine

Reply:
So instead of how requiring it in the `require-dev` section ?

Reply:
Technically legit, but no one is supposed to use dev-master now so it's not a real issue. This is mostly to make CI work with lowest dependencies as well (too bad there is no `conflict-dev` section).

Reply:
It doesn't make any sense to have a conflict-dev, as if you install the dev dependencies, it means you cloned the doctrine repo and working directly on it (you can't install dev dependencies of a dependency...). So making it into the dev section of the requirement and removing the conflict makes sense IMO.

Reply:
> So instead of how requiring it in the `require-dev` section ?

Nope, let's please not do that.

Users will be perfectly fine as long as they upgrade all their dependencies **as they should be doing**